src/pages/account/types/system/system.vue

```diff
@@ -18,13 +18,29 @@ import SettingAccountGroup from "@/pages/account/components/account-group.vue";
 
 const props = defineProps<{
   account: AccountModel;
+  transactionsCount: number;
 }>();
 const router = useRouter();
 
 const { addSuccessNotification, addErrorNotification } = useNotificationCenter();
 const accountsStore = useAccountsStore();
 const confirmAccountName = ref("");
 
+const alertDescription = computed(() => {
+  if (props.transactionsCount > 0) {
+    return {
+      message: `This action cannot be undone.`,
+      boldText: `You have ${props.transactionsCount} transactions associated with this account, they will also be deleted.`,
+      details: "Do you really want to delete this account?",
+    };
+  }
+  return {
+    message: "This action cannot be undone. You have ",
+    boldText: "You have zero transactions associated.",
+    details: " Do you really want to delete this account?",
+  };
+});
+
```

Comment:
You have slot now, pass data directly

Reply:
You need all text in bold or specific about transactions?

Reply:
We need everything as it is right now, but using slots.
Also, there're bugs:
1. You have You have
<img width="516" alt="image" src="https://github.com/user-attachments/assets/85dc0d7d-7bca-4bf1-b911-80dc6164ff3a" />

2. `description` prop right now is not used at all, so it won't be rendered.
3. Without passing `description` prop, slot won't be rendered either.

Read comments above, some of these bugs are already listed

Fixed everything in my next commits